guides/v2.1/install-gde/bk-install-guide.md

```diff
@@ -17,16 +17,16 @@ functional_areas:
   - Setup
 ---
 
-## Magento software installation
-
-Hi, we're glad you're among the 240,000 merchants worldwide who put their trust in our eCommerce software. We've gathered some information to help you get started with Magento and with your Magento installation.
+You are among the 240,000 merchants worldwide who put their trust in our eCommerce software. We've gathered some information to help you get started with Magento and with your Magento installation.
```

Comment:
Since you link to the 2.3 product availability page, I think you need to break the symbolic link for 2.3.